apps/web/sdk/index.ts

```diff
@@ -2,42 +2,76 @@ import { PlentysystemsModuleType, plentysystemsModule, client } from '@plentymar
 import { initSDK, buildModule } from '@vue-storefront/sdk';
 
 // Maintain a reference to the interceptor
-let interceptorId: number | null = null;
+let interceptorIdReq: number | null = null;
```

Comment:
We need to do some refactoring here to move code out of the file. But lets do it in a follow up task